lib/seafoam/formatters/formatters.rb

```diff
@@ -0,0 +1,8 @@
+module Seafoam
+  # Formatters are the mechanism by which `seafoam` command output is presented to the user.
+  module Formatters
+    autoload :Base, 'seafoam/formatters/base'
```

Comment:
This is always needed isn't it?

Reply:
It's not needed for `--help` or `--version`, but those are limited cases. I can unconditionally require if you'd prefer.